src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  function add(n) {
+    calculator.result += n;
+  }
+
+  function subtract(n) {
+    calculator.result -= n;
+  }
+
+  function multiply(n) {
+    calculator.result *= n;
+  }
```

Comment:
[CODE KNOWLEDGE] Use the method shorthand syntax for defining methods within an object to improve code clarity.

BAD EXAMPLE:
 multiply: function(n) {
   this.result *= n;
 },

GOOD EXAMPLE:
 multiply(n) {
   this.result *= n;
 },